build.gradle

```diff
@@ -96,21 +98,21 @@ dependencies {
   implementation 'org.apache.logging.log4j:log4j-slf4j-impl:2.14.1' // log4j -> slf4j
 
   // pbf parser
-  implementation group: 'com.github.johanneshiry', name: 'OSMonaut', version: 'v1.1.1'
+  implementation 'com.github.johanneshiry:OSMonaut:v1.1.1'
 
   // javafx
-  implementation group: 'org.openjfx', name: 'javafx-fxml', version: fxVersion
-  implementation group: 'org.openjfx', name: 'javafx-swing', version: fxVersion
-  implementation group: 'org.openjfx', name: 'javafx-graphics', version: fxVersion
-  implementation group: 'org.openjfx', name: 'javafx-web', version: fxVersion
+  implementation "org.openjfx:javafx-fxml:$fxVersion"
+  implementation "org.openjfx:javafx-swing:$fxVersion"
+  implementation "org.openjfx:javafx-graphics:$fxVersion"
+  implementation "org.openjfx:javafx-web:$fxVersion"
```

Comment:
Do we still needs this? I think its droppable as we do not want to have fx capabilities here anymore don't we? 
If this is only required for backward compatibility you may add these dependencies on the todo list in #76

Reply:
We don't need it. But I was bother by the kind of notation. ^^